src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_IndexerAccess.cs

```diff
@@ -131,15 +128,15 @@ private BoundExpression VisitIndexerAccess(BoundIndexerAccess node, bool isLeftO
 
                 // We have already lowered each argument, but we may need some additional rewriting for the arguments,
                 // such as generating a params array, re-ordering arguments based on argsToParamsOpt map, inserting arguments for optional parameters, etc.
```

Comment:
This isn't true now right? We havn't actually lowered the args yet.

Reply:
Right. I thought I got all of these, but looks like I missed one.